lib/drivers/bayerContourNext.js

```diff
@@ -416,7 +441,7 @@ module.exports = function (config) {
           }
         dataToPost.push(smbg);
       }
-    }else{
+    } else {
       debug('Device has no records to upload');
       throw(new Error('Device has no records to upload'));
```

Comment:
There's a try/catch around the calling function now, so this exception will be surfaced in the UI.